addons/mail/models/mail_mail.py

```diff
@@ -725,7 +727,7 @@ def _send(self, auto_commit=False, raise_exception=False, smtp_session=None, ali
                             value = '. '.join(e.args)
                         raise MailDeliveryException(value)
                     raise
-
-            if auto_commit is True:
+            self.env['ir.cron']._notify_progress(1)
+            if auto_commit is True:  # See if we should not commit less often ? like every 50/100 mails ?
```

Comment:
Don't understand your comment, we already make iterative steps that is configurable through ICP.

Reply:
:+1: Comment was to not forget to ask, will remove it :D